SQL_module/SubjectForm_Task2/db_manipulation.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+    include "./db_connection.php";
+
+    $table_name = "electives";
+
+    function insertInTable($title, $description, $lecturer){
+        $connection = db_connection();
+        $sql = "INSERT INTO electives(title,description,lecturer) VALUES (:title, :description, :lecturer)";
+
+        $preparedSql = $connection->prepare($sql) or die("Failed to prepare sql query.");
+        $preparedSql->bindParam(':title', $title);
+        $preparedSql->bindParam(':description', $description);
+        $preparedSql->bindParam(':lecturer', $lecturer);
+        
+        $preparedSql->execute() or die("Filed to execute sql query.");
+    }
```

Comment:
You can suuround this function body with a try/catch block as well just in case an error occurs.